meinberlin/apps/projects/templatetags/meinberlin_project_tags.py

```diff
@@ -33,6 +35,24 @@ def is_external(project):
     )
 
 
+@register.simple_tag(takes_context=True)
+def get_sorted_modules(context):
+    project = context["project"]
+    module = context.get("module", None)
+    module_qs = project.modules
+
+    if module:
+        module_qs = module.other_modules
```

Comment:
@vellip I don't understand what's happening here, when there is module in the context, we don't care about the project's modules?

Reply:
@m4ra When there is a  module in context we only care about the other modules and
not the current module, because the user is already on that module detail
page in that case.